examples/js/utils/BufferGeometryUtils.js

```diff
@@ -589,19 +589,9 @@ THREE.BufferGeometryUtils = {
 
 			var name = attributeNames[ i ];
 			var oldAttribute = geometry.getAttribute( name );
-			var attribute;
 
 			var buffer = new oldAttribute.array.constructor( attrArrays[ name ] );
-			if ( oldAttribute.isInterleavedBufferAttribute ) {
-
-				attribute = new THREE.BufferAttribute( buffer, oldAttribute.itemSize, oldAttribute.itemSize );
```

Comment:
> attribute = new THREE.BufferAttribute( buffer, oldAttribute.itemSize, oldAttribute.itemSize );

The third parameter was actually wrong. It should be `oldAttribute.normalized`. Maybe a copy/paste error.

Reply:
Awesome thanks for the catch!